.github/workflows/pythonpackage.yml

```diff
@@ -4,20 +4,27 @@ on: [push]
 
 jobs:
   build:
-
+    env:
+      DATABASE_URL: 'postgres://postgres:postgres@localhost:5432/aalondon' 
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.7]
-
     services:
       postgres:
-        image: postgres:11
+        image: postgres:10.8
```

Comment:
Any reason for downgrading Postgres image ?

Reply:
Upgraded 👍